styles/variables-silicon.less

```diff
@@ -223,6 +231,17 @@
 @agate-switch-icon-position-left: -9px;
 @agate-switch-selected-icon-position-left: ((@agate-switch-width - @agate-switch-height) + @agate-switch-icon-position-left);
 
+// ThumbnailItem
+// ---------------------------------------
+@agate-thumbnailitem-line-height: 40px;
+@agate-thumbnailitem-height: 120px;
+@agate-thumbnailitem-border: none;
+@agate-thumbnailitem-border-focus: none;
+@agate-thumbnailitem-border-width: 1px;
+@agate-thumbnailitem-content-font-size: 28px;
```

Comment:
If possible, we want to use a multiple of 3. Please check https://github.com/enactjs/agate/pull/253#discussion_r431506007 discussion.